TLM/TLM/State/Options.cs

```diff
@@ -115,46 +116,15 @@ public class Options : MonoBehaviour {
         }
 
         public static void MakeSettings(UIHelperBase helper) {
-            // tabbing code is borrowed from RushHour mod
-            // https://github.com/PropaneDragon/RushHour/blob/release/RushHour/Options/OptionHandler.cs
-            UIHelper actualHelper = helper as UIHelper;
-            UIComponent container = actualHelper.self as UIComponent;
-
-            UITabstrip tabStrip = container.AddUIComponent<UITabstrip>();
-            tabStrip.relativePosition = new Vector3(0, 0);
-            tabStrip.size = new Vector2(container.width - 20, 40);
-
-            UITabContainer tabContainer = container.AddUIComponent<UITabContainer>();
-            tabContainer.relativePosition = new Vector3(0, 40);
-            tabContainer.size = new Vector2(container.width - 20, container.height - tabStrip.height - 20);
-            tabStrip.tabPages = tabContainer;
-
-            int tabIndex = 0;
-
-            // GENERAL
-            OptionsGeneralTab.MakeSettings_General(tabStrip, tabIndex);
-
-            // GAMEPLAY
-            ++tabIndex;
-            OptionsGameplayTab.MakeSettings_Gameplay(tabStrip, tabIndex);
-
-            // VEHICLE RESTRICTIONS
-            ++tabIndex;
-            OptionsVehicleRestrictionsTab.MakeSettings_VehicleRestrictions(tabStrip, tabIndex);
-
-            // OVERLAYS
-            ++tabIndex;
-            OptionsOverlaysTab.MakeSettings_Overlays(tabStrip, tabIndex);
-
-            // MAINTENANCE
-            ++tabIndex;
-            OptionsMaintenanceTab.MakeSettings_Maintenance(tabStrip, tabIndex);
-
-            // KEYBOARD
-            ++tabIndex;
-            OptionsKeybindsTab.MakeSettings_Keybinds(tabStrip, tabIndex);
-
-            tabStrip.selectedIndex = 0;
+            //ExtUITabstrip.Test.OnSettingsUI(helper);return;
```

Comment:
remove or add new preprocessor directive before merge

Reply:
I will just remove the commented out code. Its only for testing anyway.